server.js

```diff
@@ -39,6 +42,7 @@
     var mime = express.static.mime;
     mime.define({
         'application/json' : ['czml', 'json', 'geojson', 'topojson'],
+        'application/octet-stream' : ['b3dm', 'i3dm', 'pnts', 'cmpt'],
```

Comment:
Oh actually these were added to master in the 3D Tiles PR.
However, my adding them made a duplicate copy here (I was working on master). I'll remove it.